cleanlab/count.py

```diff
@@ -67,34 +69,24 @@ def num_label_issues(
     estimation_method: str = "off_diagonal",
     multi_label: bool = False,
 ) -> int:
-    """Estimates the number of label issues in the `labels` of a dataset. Use this method to get the most accurate
-    estimate of number of label issues when you don't need the indices of the label issues.
+    """Estimates the number of label issues in a classification dataset. Use this method to get the most accurate
+    estimate of number of label issues when you don't need the indices of the examples with label issues.
 
     Parameters
     ----------
     labels :
-      An array of shape ``(N,)`` of noisy labels, i.e. some labels may be erroneous.
-      Elements must be in the set 0, 1, ..., K-1, where K is the number of classes.
-      For multi-label classification where each example can belong to multiple classes,
-      refer to documentation for this argument in :py:func:`find_label_issues <cleanlab.filter.find_label_issues>` for details.
+      Given class labels for each example in the dataset, some of which may be erroneous,
```

Comment:
Is it clear enough to link param documentation if that documentation comes from a different file (like here linking cleanlab.filter from cleanlab.count) without further explanation?

Reply:
One idea is to fully redefine all parameters once in a file for a big function like this one and then link them for the rest of the documentation in this file.

Reply:
I think it's ok to link function from different module when that is one of the main functions in the package. I imagine everybody using cleanlab knows the `find_label_issues` function. So when they are using another function, they'll be happy to hear they can just have their inputs in the same format for the famous function they already know.  I find it less ideal to ask them to refer to another function they may have never heard of, even if this function is in the same module. 

Anyway clicking on the link to the function should directly take them to the appropriate docstring in docs.cleanlab.ai, so having the function be in different file does not matter much when browsing this website.